### DISCUSSION TITLE
How is the tooltipText property of amchart line charts used in lowdefy？

### DISCUSSION THREAD

**Discussion by 15602322852** [lowdefy/lowdefy#1402] [Questions and Help]
How is the tooltipText property of amchart line charts used in lowdefy？

**Comment by Gervwyk** [comment]
Hi @15602322852 
Please provide example config, and see https://www.amcharts.com/docs/v4/concepts/tooltips/

**Reply by 15602322852** [reply]
tooltipText How to Set ？


  blocks:
          - id: my_chart
            type: AmChartsXY
            properties:
              radius: percent(80)
              height: 300 
              data: 
                _state: deviceStatusData
              xAxes:
                - type: CategoryAxis 
                  gridColor: '#FFFFFF'
                  title:  # X轴标题
                    text:  执行时间
                    fill: "#fff" # 标题颜色
                  dataFields:
                    category: 执行时间
                  
                  renderer:
                    line: # 设置X轴的线
                      strokeOpacity: 1
                      strokeWidth: 1
                      stroke: "#3787ac" 
                    grid:
                      disabled: true
                      # stroke: '#FFF'
                    minGridDistance: 20
                    labels:
                      config: # X轴字体颜色
                        fill: '#fff'
              yAxes:
                - type: ValueAxis
                  axisColor:
                    fill: '#FF6600'      
                  title: # Y轴标题
                    text:  稼动率 %
                    fill: "#fff" # 标题颜色
                  renderer:
                    minWidth: 60
                    line:  # 设置y轴的线
                      strokeOpacity: 1
                      strokeWidth: 1
                      stroke: "#3787ac"
                    grid: 
                      template:
                        strokeOpacity: 0.07
                    labels:
                      config: # Y轴字体颜色
                          fill: '#fff'
              series:
                - type: LineSeries  # LineSeries
                  name: 稼动率
                  dataFields:
                    categoryX: 执行时间
                    valueY: 稼动率
                  tooltipText: "稼动率: {valueY}%"
